### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by tomasfabry**

### CODE REVIEW

File: client/src/components/content/Content.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      selectedTag,
+      loading: props.loading,
+    })
+  }
+
+  render() {
+    const {emoji, reports, users, channels, selectedTag, loading} = this.state
+    console.log('XXX')
+    console.log(selectedTag)
+    console.log(selectedTag === undefined)
```
Comment (paldys) [inline client/src/components/content/Content.js]: Debugging stuff was left here.

Comment (hleumas) [reply client/src/components/content/Content.js]: Thanks for pointing that out. Could you please either rewrite or fix this PR?

Comment (paldys) [reply client/src/components/content/Content.js]: :+1: Done.

**Comment by paldys** [comment]
Let's close this PR in favor of #19.

**Pull request closed without merging**